src/app/shared/entry/doi/manage-dois/manage-dois-dialog.component.html

```diff
@@ -87,6 +87,41 @@ <h1 mat-dialog-title>Manage DOIs</h1>
                 <app-doi-badge *ngIf="doiInfo.value.conceptDoi" [doi]="doiInfo.value.conceptDoi" [displayInitiator]="false"></app-doi-badge>
               </span>
             </div>
+
+            <div class="mt-3" *ngIf="doiInfo.key === DoiInitiatorEnum.DOCKSTORE">
+              <div class="size-small gray-caption">
+                You can edit the metadata of a published Dockstore DOI at anytime.
+                <button mat-button (click)="toggleShowMoreEditLinkInfo()" class="private-btn">
+                  <mat-icon class="mat-icon-sm">{{ showMoreEditLinkInfo ? 'keyboard_arrow_up' : 'keyboard_arrow_down' }}</mat-icon
+                  >{{ showMoreEditLinkInfo ? 'Collapse' : 'Show more' }}
+                </button>
+              </div>
+              <div *ngIf="showMoreEditLinkInfo">
+                <p class="size-small gray-caption">
+                  Dockstore uses Zenodo's link sharing feature to create a secret link with edit access so you can edit the DOI. You must be
+                  logged into your Zenodo account when you open the link. When you are done editing the DOI, delete the link on Dockstore.
+                </p>
+                <span *ngIf="dockstoreDoiAccessLink">
+                  <a
+                    [href]="
+                      doiInfo.value.versionDoi?.name ?? doiInfo.value.conceptDoi.name | zenodoAccessLink: dockstoreDoiAccessLink.token
+                    "
+                    target="_blank"
+                    rel="noopener noreferrer"
+                  >
+                    Edit DOI on Zenodo<mat-icon class="mat-icon-sm">open_in_new</mat-icon>
```

Comment:
Ok with this as a title, but when explaining more, kinda prefer record or deposit vs DOI